test/versioned/langchain/runnables.tap.js

```diff
@@ -99,39 +98,45 @@ tap.test('Langchain instrumentation - runnable sequence', (t) => {
     'should create langchain events for every invoke call on chat prompt + model + parser',
     (t) => {
       const { agent, prompt, outputParser, model } = t.context
-
+      const api = helper.getAgentApi()
+      agent._contextManager.setContext({ initial: true })
       helper.runInTransaction(agent, async (tx) => {
-        const input = { topic: 'scientist' }
-        const options = { metadata: { key: 'value', hello: 'world' }, tags: ['tag1', 'tag2'] }
-
-        const chain = prompt.pipe(model).pipe(outputParser)
-        await chain.invoke(input, options)
-
-        const events = agent.customEventAggregator.events.toArray()
-
-        const langchainEvents = filterLangchainEvents(events)
-        const langChainMessageEvents = filterLangchainEventsByType(
-          langchainEvents,
-          'LlmChatCompletionMessage'
-        )
-        const langChainSummaryEvents = filterLangchainEventsByType(
-          langchainEvents,
-          'LlmChatCompletionSummary'
-        )
-
-        t.langchainSummary({
-          tx,
-          chatSummary: langChainSummaryEvents[0]
-        })
-
-        t.langchainMessages({
-          tx,
-          chatMsgs: langChainMessageEvents,
-          chatSummary: langChainSummaryEvents[0][1]
+        api.withLlmCustomAttributes({ 'llm.contextAttribute': 'someValue' }, async () => {
+          const input = { topic: 'scientist' }
+          const options = { metadata: { key: 'value', hello: 'world' }, tags: ['tag1', 'tag2'] }
+
```

Comment:
Instead of changing the intention of this existing test, please add a new test that exercises the new functionality specifically.